tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -410,6 +410,15 @@ ecmp/inner_hashing/test_wr_inner_hashing_lag.py:
       - "asic_type not in ['mellanox']"
       - "'dualtor' in topo_name"
 
+ecmp/test_ecmp_sai_value.py:
+  skip:
+    reason: "Only support Broadcom T1 topology with 20230531.10 or later image"
+    conditions:
+      - "topo_type not in ['t1']"
+      - "asic_type not in ['broadcom']"
+      - "release not in ['202305']"
```

Comment:
We should have this on later ones also, like 202311. 
I think we should check t0 also.

Reply:
hi @ZhaohuiS I mean there would be new release, like 202311 and 202405.